### DISCUSSION TITLE
.NET Core support

### DISCUSSION THREAD

**Discussion by jkuek** [awslabs/aws-solutions-constructs#502] [General]
Hi, any plans to make this library available on .NET Core?

**Comment by diesal11** [comment]
Looks like it's been added to the build #426, but not yet published to Nuget.

@hnishar @biffgaut Is someone able to provide some clarity here?

**Comment by biffgaut** [comment]
The technology is there, it's the processes/policies around publishing to NuGet that has us blocked. There is not current ETA on this, although we may try to navigate the challenges again this Fall.

If you're feeling ambitious, you could try doing a build of the library according to the Building Patterns section of CONTRIBUTING.md. In the dist folder you should find all the artifacts we publish to each registry, including what we would publish to NuGet. You may be able to add these to your project directly. I doubt we'd be able top provide much support in this area though.

**Reply by biffgaut** [reply]
There's a dist folder under every construct folder with the nuget packages for that construct.

**Reply by diesal11** [reply]
Thanks, we ended up grabbing the `.nuget` files from the release assets and pushing them to a private repo for now.

**Comment by biffgaut** [comment]
Please keep us posted on how that works, it would be great to have that feedback so we can address any glitches prior to overcoming our current hurdles.
